src/libertem/analysis/gridmatching.py

```diff
@@ -221,21 +218,19 @@ def fastmatch(cls, correlation_result: CorrelationResult, zero, a, b, parameters
     def full_match(cls, correlation_result: CorrelationResult, zero, cand=[], parameters={}):
```

Comment:
Using mutable objects as default value means they are shared between calls. The best practice here is to use None instead, check with `is None` and initialize with a default value in the function body.